tests/python/contrib/test_hexagon/unit_tests.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
```

Comment:
This seems to be a merge issue since this file has been removed. Please remove this file.

Reply:
Done.